src/main/java/com/zufar/icedlatte/order/api/OrderApi.java

```diff
@@ -0,0 +1,23 @@
+package com.zufar.icedlatte.order.api;
+
+import com.zufar.icedlatte.openapi.dto.*;
+import java.util.List;
+
+public interface OrderApi {
```

Comment:
You do not need to create this interface. Delete it.

Reply:
Removed in [46e65b4](https://github.com/Sunagatov/Iced-Latte/pull/217/commits/46e65b4c9646349a131383ace42afdeddba4b663)

Just curious: what about src/main/java/com/zufar/icedlatte/cart/api/CartApi.java?